addons/mail/models/mail_thread.py

```diff
@@ -1867,8 +1867,8 @@ def _message_post_after_hook(self, message, msg_vals):
     # MESSAGE POST TOOLS
     # ------------------------------------------------------
 
-    def message_post_with_view(self, views_or_xmlid, **kwargs):
-        """ Helper method to send a mail / post a message using a view_id to
+    def _send_message_with_view(self, views_or_xmlid, message_log=False, **kwargs):
```

Comment:
Mmmh ... to check later but I think we could use mail render mixin tools instead.